meteor/client/ui/SegmentTimeline/SegmentTimeline.tsx

```diff
@@ -748,6 +806,15 @@ export class SegmentTimelineClass extends React.Component<Translated<IProps>, IS
 		}
 	}
 
+	renderEditorialLine() {
+		if (this.props.budgetDuration !== undefined) {
+			let lineStyle = {
+				left: this.props.budgetDuration * this.props.timeScale - this.props.scrollLeft * this.props.timeScale + 'px',
```

Comment:
Shouldn't this be just `(this.props.budgetDuration - this.props.scrollLeft) * this.props.timeScale + 'px'` ?